dolfyn/velocity.py

```diff
@@ -25,7 +25,7 @@ class Velocity():
     ########
     # Major components of the dolfyn-API
 
-    def rotate2(self, out_frame='earth'):
+    def rotate2(self, out_frame='earth', inplace=True):
```

Comment:
Whatever we do for the functions in `rotate.api`, we definitely `inplace=True` for all of these methods.